verify-service/src/main/java/io/inji/verify/dto/authorizationRequest/AuthorizationRequestResponseDto.java

```diff
@@ -0,0 +1,35 @@
+package io.inji.verify.dto.authorizationRequest;
```

Comment:
package names in java are not usually followed as camel case, it should be authorizationrequest, please follow this all places.

Reply:
Done